File: pertemuan 11/tugas.php
<?php
$nim = readline("Masukkan NIM: ");

$digit = [];
for ($i = 0; $i < strlen($nim); $i++) {
    $digit[] = intval($nim[$i]);
}

$total = 0;
foreach ($digit as $d) {
    $total += $d;
}

$max_val = $digit[0];
foreach ($digit as $d) {
    if ($d > $max_val) {
        $max_val = $d;
    }
}

$minim = $digit[0];
foreach ($digit as $d) {
    if ($d < $minim) {
        $minim = $d;
    }
}

$rata = $total / count($digit);

$rev = [];
for ($i = count($digit) - 1; $i >= 0; $i--) {
    $rev[] = $digit[$i];
}

echo "digit         : ";
print_r($digit);
echo "total         : $total\n";
echo "max           : $max_val\n";
echo "min           : $minim\n";
echo "rata-rata     : $rata\n";
echo "reverse array : ";
print_r($rev);
?>

File: pertemuan 11/tugas.py
nim = input("Masukkan NIM: ")

digit = [int(c) for c in nim]

total = 0
for d in digit:
    total += d

max_val = digit[0]
for d in digit:
    if d > max_val:
        max_val = d

minim = digit[0]
for d in digit:
    if d < minim:
        minim = d

rata = total / len(digit)

rev = []
for i in range(len(digit)-1, -1, -1):
    rev.append(digit[i])

print("digit        :", digit)
print("total        :", total)
print("max          :", max_val)
print("min          :", minim)
print("rata-rata    :", rata)
print("reverse array:", rev)
